translations/de/module_de_lang.php

```diff
@@ -52,7 +53,7 @@
     'wd_config_message' => 'Ihre Nachricht',
     'wd_config_payment_action' => 'Zahlungsaktion',
     'wd_config_payment_action_desc' => 'Wählen Sie "Buchung", um automatisch eine Buchung durchzuführen, oder "Genehmigung", um eine manuelle Buchung zu ermöglichen.',
-    'wd_config_payolution_terms_url' => 'payolution URL',
+    'wd_config_payolution_terms_url' => 'Payolution URL',
```

Comment:
Is this approved from @hbrandl that payolution is written in Capitalazed?

Reply:
it was discussed here: https://wd-cee.slack.com/archives/GFZKK7SCA/p1559573664024800?thread_ts=1559051638.019400&cid=GFZKK7SCA